draft-ietf-quic-transport.md

```diff
@@ -4742,6 +4781,22 @@ active_connection_id_limit (0x0e):
   When a zero-length connection ID is being used, the active_connection_id_limit
   parameter MUST NOT be sent.
 
+handshake_connection_id (0x0f):
+
+: The value that the endpoint included in the Source Connection ID field of the
+  first Initial packet it sends during the handshake. Endpoints MUST validate
+  that this transport parameter is present and that it matches the value that
+  was received in Initial packets. Authenticating this value ensures that an
+  attacker is unable to influence the selection of connection IDs during the
```

Comment:
to avoid jinxing things, how about `Authenticating this value mitigates an attacker's ability to influence the selection of connection IDs during the handshake.`

Reply:
What if some of these CIDs are zero length? Do we omit them?

Reply:
My read of the zero-length CID case is that the TP MUST be included, and be zero-length. Perhaps we should add a sentence to clarify that.

Reply:
Done! Thanks both.

As for the "mitigates" formulation, I think that we are safe in making claims, but I realize that we don't need to.  I've removed a lot of text from the transport parameter definitions and replaced it with a pointer to the new section.